autoload/clap/provider/windows.vim

```diff
@@ -11,24 +11,45 @@ function! s:jump(t, w) abort
   execute a:w.'wincmd w'
 endfunction
 
-function! s:format_win(tab, win, buf) abort
-  let modified = getbufvar(a:buf, '&modified')
-  let name = bufname(a:buf)
+function! s:get_clap_winids() abort
+	let clap_winids = []
+	for clap_win in ['display', 'input', 'spinner', 'preview']
+		let clap_win_var = 'g:clap.' . clap_win . '.winid'
+		if exists(clap_win_var)
+			call add(clap_winids, g:clap[clap_win].winid)
+		endif
+	endfor
```

Comment:
```suggestion
let clap_winids = map(filter(['display', 'input', 'spinner', 'preview'], 'exists("g:clap.".v:val.".winid")'), 'g:clap[v:val].winid')
```

Reply:
Thanks, that's going to be useful—didn't know (N)vim could do map.